tests/ui/simd/intrinsic/float-minmax-pass.rs

```diff
@@ -3,17 +3,14 @@
 
 // Test that the simd_f{min,max} intrinsics produce the correct results.
 
-#![feature(repr_simd, platform_intrinsics)]
+#![feature(repr_simd, core_intrinsics)]
 #![allow(non_camel_case_types)]
 
 #[repr(simd)]
 #[derive(Copy, Clone, PartialEq, Debug)]
 struct f32x4(pub f32, pub f32, pub f32, pub f32);
 
-extern "platform-intrinsic" {
-    fn simd_fmin<T>(x: T, y: T) -> T;
-    fn simd_fmax<T>(x: T, y: T) -> T;
-}
+use std::intrinsics::simd::*;
```

Comment:
I'm assuming the reason only some of these got converted to imports in tests is that it's annoying to do with a search and replace?

Reply:
Yeah, I changed this one since @bjorn3 mentioned it but most of the diff in tests/ui is search-and-replace. it'd be tedious to convert them all and it's orthogonal to what this PR is trying to achieve.

Reply:
Oh yea, that's fine, just wanted to make sure I wasn't missing anything